projects/observability/src/shared/components/bar-gauge/bar-gauge.component.ts

```diff
@@ -162,5 +162,6 @@ interface BarSegment extends Segment {
 
 export const enum BarGaugeStyle {
   Regular = 'regular',
-  Compact = 'compact'
+  Compact = 'compact',
+  SingleBar = 'single-bar'
```

Comment:
May be naming could have been different. All of these are single bars only rights?

Reply:
Yes but the rest of the gauges have bars, references, title, etc... So in this case I have only a single bar.